simpeg_drivers/plate_simulation/sweep/options.py

```diff
@@ -122,7 +126,17 @@ def trials(self) -> list[dict]:
         """Returns a list of parameter combinations to run for each trial."""
         names = [s.name for s in self.sweeps]
         iterations = itertools.product(*[np.linspace(*s()) for s in self.sweeps])
-        return [dict(zip(names, i, strict=True)) for i in iterations]
+        options_dict = dict_mapper(self.template_options, [self.format_value])
+
+        trials = []
+        for iterate in iterations:
+            trial = dict(zip(names, iterate, strict=True))
+            trial = dict_mapper(trial, [self.format_value])
```

Comment:
The `trial` contains values that will be used in the computation, and the `dict_mapper` call here is rounding off float values.  We should only round to create the string for uuid generation.